torch/_utils_internal.py

```diff
@@ -140,7 +140,7 @@ def log_trace_structured_event(*args, **kwargs) -> None:
     pass
 
 
-def log_cache_bypass(*args, **kwargs) -> None:
+def log_cache_operation(*args, **kwargs) -> None:
```

Comment:
Can we have a more detailed signature here? Makes it easier to typecheck sites from oss only